src/App.js

```diff
@@ -1,9 +1,72 @@
 import React from 'react';
 import './App.css';
+import NavButton from "./components/NavButton";
+import Product from "./components/Product";
+import Tile from "./components/Tile";
+
+import handyBag from "./assets/bag_1.png"
+import stylishBag from "./assets/bag_2.png"
+import simpleBag from "./assets/bag_3.png"
+import trendyBag from "./assets/bag_4.png"
+import brand from "./assets/brand.png"
+import story from "./assets/our_story.png"
+
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
+      <>
+        <h1>Handbags & Purses</h1>
+        <nav>
+            <NavButton
+                buttonName={"to the collection"}
+            />
+            <NavButton
+                buttonName={"shop all bags"}
+            />
+            <NavButton
+                buttonName={"pre-orders"}
+                disabled={true}
+            />
+        </nav>
+        <main>
+            <Product
+                redLabel={"Best seller"}
+                image={handyBag}
+                bagName={"The handy bag"}
+                price={"€400,-"}
```

Comment:
Hey Michiel! Wat leuk dat ik jouw code mag bekijken :)
Hier even een comment over de alinea: 
de euro teken en the comma en middenstreepje zou je (volgens de aandachtspuntenlijst van Nova) in de component.js moeten zetten. Dus zo in jouw App.js`<h4>€{price},-</h4>`                                    
In jouw Product.js`€{price},-</h4>`
Bij de andere elements ga ik dit niet nog een keer herhalen - anders ziet je hier met onnodige comments ;)